Add a bio file for izike

izike opened the pull request:
Add a bio file #39

nbessa committed:
Add a bio file for nbessa

ericblow committed:
Add a bio file for Eric Blow

Blah Blah Blah

githubteacher committed:
Add a bio file for GitHub Teacher

nukdcbear committed:
Add a bio file for GitHub nukdcbear

ThatOneCloud committed:
Add a bio for Thatonecloud

Optional extended description

mikercline committed:
Add a bio file for mikercline

Creates a bio file for mikercline that includes information about the user.

WubbaDuck committed:
Add a bio file for WubbaDuck

jasonmassey committed:
Add bio file for jasonmassey

nicban2008 committed:
add a bio file for nicban2008

nbessa committed:
Update nbessa.md

fry-0 committed:
fry-0 bio

Commit:
Created a new file

Commit:
Fixed an issue, = was changed to -

daltsi committed:
Create daltsi.md

JackMcRoberts committed:
Create JackMcRoberts.md

DanielCrispino committed:
Add a bio for Daniel crispino

ThatOneCloud committed:
Merge pull request #29 from githubteacher/Thatonecloud-bio

Add a bio for Thatonecloud

stevensgill committed:
Update README.md

stevensgill committed:
Create stevegill.md

ericblow commented on docs/Thatonecloud.md:
```diff
@@ -0,0 +1,3 @@
+# demo
+
+- Not typing a lot on a phone...
```
You forgot to add your Name, Position and Location. But I guess that is hard working from a phone. :smile:

githubteacher committed:
Specify the correct San Francisco for GH Teacher

WubbaDuck committed:
Add last name

DanielCrispino committed:
Update Danielcrispino.md

JackMcRoberts committed:
Made title, location more specific

nicban2008 committed:
Merge pull request #26 from githubteacher/nicban2008

add a bio file for nicban2008
this is my merge

Commit:
Added additional information into my occupation

nicban2008 committed:
Revert "add a bio file for nicban2008"

githubteacher committed:
Merge pull request #24 from githubteacher/githubteacher-bio

Fixes #3

ericblow committed:
Merge pull request #40 from githubteacher/ericblow-bio

Fixes #17

mikercline committed:
Merge pull request #32 from githubteacher/mikercline-bio

Fixes #22

jasonmassey committed:
Merge pull request #25 from githubteacher/jasonmassey-bio

Fixes #7 Add bio file for jasonmassey

nukdcbear committed:
Merge pull request #28 from githubteacher/nukdcbear-bio

Fixes #13

WubbaDuck committed:
Merge pull request #27 from githubteacher/wubbaduck-bio

Fixes #9

daltsi committed:
Merge pull request #38 from githubteacher/daltsi-bio

fixes #16

nicban2008 committed:
Merge pull request #43 from githubteacher/revert-26-nicban2008

add a bio file for nicban2008
fixes

stevensgill committed:
Merge pull request #42 from githubteacher/stevensgill-bio

Stevensgill bio

Commit:
Merge pull request #30 from githubteacher/rnewsom-bio

fixes #44

nbessa committed:
Merge pull request #36 from githubteacher/nbessa-bio

Fixes:#8

DanielCrispino committed:
Merge pull request #37 from githubteacher/Danielcrispino-bio

fixes #35

fry-0 committed:
Merge pull request #34 from githubteacher/fry-bio

Fixes #10

JackMcRoberts committed:
Merge pull request #31 from githubteacher/JackMcRoberts-bio

Create JackMcRoberts.md

nukdcbear committed:
Add a favorite vacation spot, update location and occupation.

githubteacher committed:
Add a favorite vacation spot for @githubteacher.

daltsi committed:
First commit whoooo hoooooo

JackMcRoberts committed:
Added a favorite vacation spot

Commit:
Updated my bio

Commit:
Add a favourite vacation spot

jasonmassey committed:
add favorite vacation spot for @jasonmassey

ericblow committed:
adding more details about favorite food

WubbaDuck committed:
Add favorite vacation spot for @wubbaduck.

githubteacher committed:
Merge pull request #45 from githubteacher/githubteacher-more-bio

Add a favorite vacation spot for @githubteacher

nukdcbear committed:
Merge pull request #52 from githubteacher/nukdcbear-more-bio

Add a favorite vacation spot, update location and occupation for nukdcbear

stevensgill committed:
Merge pull request #50 from githubteacher/stevensgill-more-bio

Updated my bio

jasonmassey committed:
Merge pull request #51 from githubteacher/jasonmassey-more-bio

Add favorite vacation spot for @jasonmassey

daltsi committed:
Merge pull request #48 from githubteacher/daltsi-more-bio

First commit whoooo hoooooo

nbessa committed:
Merge pull request #49 from githubteacher/nbessa-more-bio

Add a favourite vacation spot for nbessa

WubbaDuck committed:
Merge pull request #54 from githubteacher/wubbaduck-more-bio

Add favorite vacation spot for @wubbaduck.

JackMcRoberts committed:
Merge pull request #53 from githubteacher/JackMcRoberts-more-bio

Added a favorite vacation spot for Jack McRoberts

fry-0 committed:
Add vacation spot.

ericblow committed:
Merge pull request #46 from githubteacher/ericblow-morebio

Adding more details about favorite food

fry-0 committed:
Merge branch 'master' of https://github.com/githubteacher/developers-2015-11-23

michaelsbroughton committed:
Just adding a comment

This is a change made to see the different behavior when working with branches.

michaelsbroughton committed:
Add my bio file

michaelsbroughton committed:
Making changes pointed out during code review.

michaelsbroughton committed:
Merge pull request #57 from githubteacher/michaelsbroughton

Fixes #56
